src/trackers/ANIRENA.py

```
@@ -0,0 +1,411 @@
+        if meta.get('anime'):
+            audio_langs_raw = meta.get('audio_languages', [])
+            if audio_langs_raw is None:
+                audio_langs_raw = []
+            audio_langs = [l.lower() for l in audio_langs_raw if l]
+            
+            if 'japanese' in audio_langs and len(audio_langs) == 1:
+                # If it's only Japanese audio, check subs
+                sub_langs_raw = meta.get('subtitle_languages', [])
+                if sub_langs_raw is None:
+                    sub_langs_raw = []
+                sub_langs = [l.lower() for l in sub_langs_raw if l]
```

Ruff (0.15.12)

[error] Ambiguous variable name: `l`

(E741)